assets/js/modules/reader-revenue-manager/index.js

```diff
@@ -71,3 +79,35 @@ export const registerModule = ( modules ) => {
 		},
 	} );
 };
+
+export const registerNotifications = ( notifications ) => {
+	notifications.registerNotification( 'setup-success-notification-rrm', {
```

Comment:
The notification seems to have a little delay and does not appear immediately, not at least on a first load. Is this expected?

Reply:
I've compared the delay between the main branch and this feature branch and it seems pretty much the same in both cases. 

main branch (Legacy):

https://github.com/user-attachments/assets/4e26018c-46c7-4e3e-a0e9-3a1a58d9b234

feature branch (Refactored):

https://github.com/user-attachments/assets/760e8296-154d-4645-84e6-7aedf0d26947

Reply:
We potentially could do a few performance improvements - but like before, we are evaluating the queue on page load which is similar to rendering all notification components directly and evaluating them. So the lag shouldn't / hasn't increase here?

Reply:
Hmm, I'm not sure why, but I'm clearly seeing a significant delay when compared with the base branch.

Base branch:

https://github.com/user-attachments/assets/e9329be5-9e56-48ca-8eaa-87fdc5d2771f

Feature branch:

https://github.com/user-attachments/assets/f530cff8-8299-4985-9419-3684524ea21e

Browser session storage was cleared before both tests. I don't think it is worth holding this issue off for this matter, but it might be worth creating an issue for investigation later on. WDYT?

Reply:
I realised that we actually don't hold up evaluating the whole queue for subtle notifications in the legacy components. That explains the additional lag as now we evaluate even banner and error notifications. Maybe it might be worth creating a separate issue and seeing if we can evaluate different areas separately with the understanding that two types of notifications can appear on the page at the same time (like it used to with the legacy implementation). I have created #9488 to investigate this further.